media/js/firefox/firstrun/new-firstrun.js

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function (Mozilla) {
+    'use strict';
+    let scene = document.getElementById('scene');
```

Comment:
CircleCI is still failing when running `collectstatic` (which concatenates and minifies our JS). I'm guessing it's because there is still some ES6 variable names here and below.